node-operators/next-steps.md

```diff
@@ -227,6 +227,8 @@ mkdir -p ~/.witnet/config
 
 nano ~/.witnet/config/master.key 
 
+chmod 777 ~/.witnet/config
```

Comment:
I was about to comment that I had never any need to run this. But then I realized that all my config folders were already `drwxrwxrwx`, which probably means that actually run this at some point.

Reply:
Exactly. In the typical flow (i.e. starting a container without importing a master key), the .witnet/config is generated with appropriate permissions, so reusing that directory will always work.

It's only when you try to import a master key and have manually recreated the .witnet tree that you need to explicitly broaden those permissions. Minor thing really, but would trip up more than one.

Cheers,

Reply:
@datgous you can merge this commit into `gitbook` branch now.